xcp/cpiofile.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
Is this safe to remove?

Reply:
AFAIK, it should be:

At least in git history, `xcp/cpiofile.py` never had something that checks `__name__` like:
```py
if __name__ == __main__:
```
which would be needed in order to run it as a script, or is there any other possiblity?

Of course I can remove that from the commit if there is any doubt!

Reply:
@psafont Pushed a backout of the change to the file mode and the shebang line.

Currently, 19 of the 46 *.py files have the shebang and it's not a problem to have it,
it should just be superflous, I believe, but that's not for now.

Reply:
Python modules execute all statements when they are loaded, the `__name__` guard blocks code from being executed at module load when it's not being executed from the CLi directly. So if there are no statements that allow the module to be run as a cli program it should be safe to remove the shebang

Reply:
@psafont That's also my state of knowledge. When I get to it, I'll open a PR to change file permissions and remove the shebang for all files which don't have `__name__` guard blocks.